apps/dashboard/test/integration/support_ticket_integration_test.rb

```diff
@@ -0,0 +1,80 @@
+require 'test_helper'
+
+class SupportTicketIntegrationTest < ActionDispatch::IntegrationTest
+
+  def setup
+    Configuration.stubs(:support_ticket_enabled?).returns(true)
+    Configuration.stubs(:support_ticket_config).returns({
+      email: {
+        to: "to_address@support.ticket.com",
+      }
+    })
+    Rails.application.routes.append do
+      get "/support", to: "support_ticket#new"
+      post "/support", to: "support_ticket#create"
+    end
+    Rails.application.reload_routes!
```

Comment:
Is there a better way of doing this? ^^

As requested by Gerard, the support ticket routes are disabled if no configuration is available.
This happens very early in the Rails setup that cannot be stub when the tests run.

Is there another way @johrstrom?

Reply:
I think this is fine in a test.